app/views/exercises/readme.erb

```diff
@@ -10,9 +18,9 @@
         <a href="readme">Readme</a>
       </li>
       <li role="presentation">
-        <a href="../<%= problem.slug %>"> Test Suite</a>
+        <a href="tests"> Test Suite</a>
```

Comment:
I find both the old and the new confusing here. What do you think of putting the full relative path?